src/arb/Complex.jl

```diff
@@ -72,10 +72,6 @@ end
 
 # TODO: implement hash
 
-function check_parent(a::ComplexFieldElem, b::ComplexFieldElem)
-  return true
-end
```

Comment:
Yes I verified that the compiler produces identical code: if the inputs have identical types and the parent is a singleton then it can prove that `parent(a) === parent(b)` always is true.

Reply:
great, thanks!